src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+policies-history.cleaner.period=disabled
```

Comment:
This should be in the `src/main/application.properties` file with a `%test.` prefix:
```suggestion
%test.policies-history.cleaner.period=disabled
```

Reply:
Ah, that's smarter. Thanks :)

Reply:
Yeah both approaches work but having a single file is easier to maintain.